tests/converter/rasa_sample_project/actions.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import requests
+import json
+from rasa_core_sdk import Action
+
+logger = logging.getLogger(__name__)
+
+
+class ActionJoke(Action):
+    def name(self):
+        # define the name of the action which can then be included in training stories
+        return "action_joke"
+
+    def run(self, dispatcher, tracker, domain):
+        # what your action should do
+        request = json.loads(
+            requests.get("https://api.chucknorris.io/jokes/random").text
```

Comment:
I appreciate the humor here, but since project reflects Cisco's HR policies, I would support creating a list of sanitized jokes and responding one at random.

Reply:
You suggest we should change the sample code? Are some of the jokes inappropriate?

Reply:
I think it is fine since this is the Rasa sample project, but in case you want to be safe you should modify the rasa sample project to return a random "safe" joke.

Reply:
I can check the text of the joke and see if it contains any bad words, if it does, then I can request a new joke and repeat.